src/python/turicreate/test/test_models.py

```diff
@@ -6,6 +6,9 @@
 from __future__ import print_function as _
 from __future__ import division as _
 from __future__ import absolute_import as _
+
+# mark it as part of toolkits
+import turicreate.toolkits as _  # noqa
```

Comment:
Now that we're using pytest mark, we don't need this change?

Reply:
let it be here.

Reply:
fine, I will remove it.